Apps/frontend/src/views/Oscilloscope.svelte

```diff
@@ -15,73 +15,100 @@
 
   let isEnabled = false;
 
+  // ----- Svelte lifecycle hooks -----
   onMount(() => {
-    socket = new WebSocket(import.meta.env.VITE_BACKEND_WEBSOCKET);
+    socket = createSocket();
+
+    socket.onopen = socketOnOpen;
```

Comment:
Because we are assigning the function to the .OnSomething variable inside the socket.
If we would write it with () we would call the function and not pass it. AFAIK

Reply:
Ah, okay, thanks :)